src/json.c

```diff
@@ -28,30 +28,68 @@
  ********************************************************************************/
 
 #include "common.h"
+//#ifdef SERVER
+//#include "rapiddiskd.h"
+//#endif
 #include <jansson.h>
 
-#if !defined SERVER
-
-#include "cli.h"
-
 /*
  * JSON output format:
  * {
+ *    "message": "Optional message",
  *    "status": "Success"
  * }
  */
 
-int json_status_return(int status)
+/**
+ *
+ * This function is intended to provide a simple way to create a JSON string which contains
+ * a status ("Success" or "Failed") derived from an integer, and an optional message.
+ *
+ * \param return_value if @p return_value is != @p 'SUCCESS', the JSON result will
```

Comment:
Is this comment supposed to be @param and not \param?

Reply:
Both syntax are ok, but I changed the `\` prefix to the `@` prefix because I (hope to have) always used `@` elsewhere.

Another use of `\` is at line:

https://github.com/pkoutoupis/rapiddisk/blob/db51403ad0934462e8cd710da12bd56879808596/src/json.c#L70

Both fixed in forthcoming commit.